src/app/data/services/ajax_service.spec.ts

```diff
@@ -61,6 +61,7 @@ describe('AjaxService', function () {
         expect(parsed.content_block.api_key).toBeUndefined();
         expect(parsed.content_block.api_host).toBeUndefined();
         expect(parsed.private_api_key).toBe('pk');
+        expect(request.headers).toEqual({'X-HTTP-Method-Override': 'PUT'});
```

Comment:
Shouldn't the request headers *include* a header named `X-HTTP-Method-Override`?